### DISCUSSION TITLE
Confirmation before autolevelling

### DISCUSSION THREAD

**Discussion by blcoyote** [Jyers/Marlin#282] [Ideas]
Hi,
Currently when i fatfinger the BLTouch firmware menus and accidentally press the level button on the front screen, the auto levelling takes a long while and is uninterruptible.

Is there a way to make a confirmation screen before initiating autolevel procedures?
either in config files or is this something that should be added on a lower level?

**Comment by Cancklehead** [comment]
I was just thinking of this also today. I've hit the leveller by accident a few times. Would be cool to confirm or go back....

**Comment by Jyers** [comment]
This would be an easy enough implementation, I'll add it to the todo list

**Reply by Cancklehead** [reply]
Wow, cool. Thanks again for your hard work, I'm loving your setup!

**Comment by Schnabulation** [comment]
I just switch the printer off and on again. :-)
